cg/models/cg_config.py

```diff
@@ -141,13 +143,22 @@ class DataDeliveryConfig(BaseModel):
     covid_report_path: str
 
 
+class EmailBaseSettings(BaseModel):
+    sender_password: str
+    sender_email: EmailStr
+    sll_port: int
+    smtp_server: str
+
+
 class CGConfig(BaseModel):
     database: str
     environment: Literal["production", "stage"] = "stage"
     madeline_exe: str
     bed_path: str
     delivery_path: str
     max_flowcells: Optional[int]
+    email_base_settings: EmailBaseSettings
+
     # Base APIs that always should exist
     status_db_: Store = None
```

Comment:
why the last underscore in "status_db_"?

Reply:
I don't know :) I didn't add it!